src/code/fv-components/egg-hatch-drake.js

```diff
@@ -0,0 +1,53 @@
+import React, { PropTypes } from 'react';
+import AnimatedSprite from '../components/animated-sprite';
+import OrganismView from '../components/organism';
+import classNames from 'classnames';
+
+export default class EggHatchDrakeView extends React.Component {
+
+  static propTypes = {
+    id: PropTypes.string,
+    eggClasses: PropTypes.string,
+    drakeClasses: PropTypes.string,
+    drake: PropTypes.object,
+    hatchStarted: PropTypes.bool,
+    hatchComplete: PropTypes.bool,
+    onHatchComplete: PropTypes.func
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
The way I see it, the `hatchComplete` prop does not indicate whether a drake is hatched. In the case where the level hides the user drake, `hatchComplete` is *always* passed as false, even though the drake is sometimes hatched. This feels like a contradiction to me.

If it only seems confusing to me, then I support your decision to leave it as is!